lib/yocaml_unix/runtime.ml

```diff
@@ -106,3 +110,16 @@ let read_dir path =
   try Sys.readdir path |> Array.to_list with
   | _ -> []
 ;;
+
+let hash value =
+  let open Cryptokit in
+  value |> hash_string (Hash.sha256 ()) |> transform_string (Hexa.encode ())
+;;
+
+let content_changes path new_content =
+  let open Yocaml.Try.Monad in
+  let+ old_content = read_file path in
```

Comment:
If the file does not exists, it should returns `Ok true` (otherwise it will raise an error)

Reply:
Fixed (Well spotted :) )